app/src/main/res/values/dimens.xml

```diff
@@ -9,9 +9,13 @@
     <dimen name="text_size_character_id">25sp</dimen>
     <dimen name="margin_card_view_item">10dp</dimen>
     <dimen name="padding_card_view_item">6dp</dimen>
-    <dimen name="thumbnail_character_min_size">400dp</dimen>
+    <dimen name="card_view_thumbnail_character_min_size">400dp</dimen>
```

Comment:
did you try this in smaller devices?

Reply:
Mau, I make some changes and now it works good in a smaller device! Fixed!